egs/ami/s5/local/tfrnnlm/run_lstm.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+mic=ihm
+ngram_order=4
+stage=1
+weight=0.5
+
+. ./utils/parse_options.sh
+. ./cmd.sh
+. ./path.sh
+
+set -e
+
+dir=data/tensorflow_lstm
+mkdir -p $dir
+
+if [ $stage -le 1 ]; then
+  local/tfrnnlm/rnnlm_data_prep.sh $dir
+fi
+
+mkdir -p $dir
+if [ $stage -le 2 ]; then
+# the following script uses TensorFlow. You could use tools/extras/install_tensorflow_py.sh to install it
+  python steps/tfrnnlm/rnnlm.py --data_path=$dir --save_path=$dir/rnnlm --vocab_path=$dir/wordlist.rnn.final
```

Comment:
Since this script has lstm in its name, shouldn't it be explicitly calling the  lstm version of the script?
Please try to use "-" instead of "_" in options to python scripts, if possible, for consistency with the rest of our Python scripts.